packages/create-app/src/run.ts

```diff
@@ -79,16 +79,38 @@ export async function run(
 
   consola.info(`Copying files into project directory`);
```

Comment:
Begin actual meat change for the template generation. Just writes from a map instead of cp from a directory.